array-manipulation-2/src/includes.js

```diff
@@ -0,0 +1,22 @@
+/* exported includes */
+
+/*
+
+******************************* PSEUDOCODE *******************************
+
+- check to see if the array is empty
+  - if yes, give back false
+- look through each value in the array
+- if there is a match, give true, otherwise give back false
+
+**************************************************************************
+
+*/
+
+function includes(array, value) {
+  if (array.length === 0) return false;
```

Comment:
This can be removed. If the array is empty, the loop below would never start, which leaves you with the same result